src/containers/shared/components/Transaction/AMMClawback/TableDetail.tsx

```diff
@@ -0,0 +1,33 @@
+import { Trans, useTranslation } from 'react-i18next'
+import { Amount } from '../../Amount'
+import { Account } from '../../Account'
+import { TransactionTableDetailProps } from '../types'
+
+export const TableDetail = ({ instructions }: TransactionTableDetailProps) => {
+  const { t } = useTranslation()
+  const { amount2, amount, holder } = instructions
+  if (amount2) {
+    return (
+      <div className="ammClawback">
+        <Trans i18nKey="action_from_and">
+          <span className="label">{t('claws_back')}</span>
+          <Amount value={amount} displayIssuer />
+          and
+          <Amount value={amount2} displayIssuer />
```

Comment:
Even if the flag `tfClawTwoAssets` is enabled, only one IOU is sent back to the issuer. The other asset in the pair is sent back to the `holder` account.

The difference occurs in the behavior of `AMMWithdraw` transaction -- both the assets are proportionally withdrawn instead of a single-asset withdrawal.

Is it possible to indicate this in the wording of the explorer?

`Asset1` is clawed_back and sent to the `Issuer1` whereas `Asset2` is transferred back to the `Holder`

Reply:
I agree. Will fix this

Reply:
Hi @ckeshava, based on the description on xrpl.org for the `tfClawTwoAssets`:
`
Claw back the specified amount of Asset, and a corresponding amount of Asset2 based on the AMM pool's asset proportion; both assets must be issued by the issuer in the Account field. If this flag isn't enabled, the issuer claws back the specified amount of Asset, while a corresponding proportion of Asset2 goes back to the Holder.
`
I think both asset would be clawed back to the issuer with the flag enabled, otherwise there's no need for the condition that the second asset has to be issued by the issuer as well. I think my initial wording is correct here

Reply:
The exact behavior depends on whether the second asset was issued by the `Issuer`. I don't remember the initial wording in your PR. We can use the language from the official documentation verbatim.